src/graph.ts

```diff
@@ -454,4 +455,44 @@ export class Graph {
       (res) => res.body.graph
     );
   }
+
+  addVertexToVertexCollection(
```

Comment:
@dlozina-macrometa I think this already exists  with graph.vertexCollection(collectionName)
      .save(...); (we have a save function which is doing this I believe)

Reply:
@koshy-thomas yes! You are right. I wanted to change this method for easier use. To be honest, save was confusing for me. But if we expose this new method, we are backward compatible. People can use `chain` method for collection or just call `addVertexToVertexCollection`. What do you think?
We already did similar things on a couple of methods in our SDK, for example, `createGraph` method is an abstraction of `const graph = this.graph(graphName).create(properties);` (Removed necessary chaining to make it simple). I like this approach.